datarobot_batch_scoring/reader.py

```diff
@@ -432,6 +432,19 @@ def sniff_dialect(sample, encoding, sep, skip_dialect, ui):
     return dialect
 
 
+def get_opener_and_mode(is_gz, text=False):
```

Comment:
I'm not sure it's going to work for gzipped japanese dataset. I think we need some sort of combination for is_gz and text.

Reply:
in PY3 gzip.open has encoding parameter. For PY2 please take a look here: https://github.com/datarobot/batch-scoring/pull/161/files#diff-6c240e73b54162e5fce6a481761017a2R527